webpack.config.js

```diff
@@ -93,15 +94,61 @@ function getRulesConfig(env) {
   ]);
 }
 
-function getAllPossibleLanguages() {
+function getAllConfigs() {
   const srcDirectory = 'app/configurations';
   return fs.readdirSync(srcDirectory)
     .filter(file => /^config\.\w+\.js$/.test(file))
-    .map(file => require('./' + srcDirectory + '/' + file).default.availableLanguages)
+    .map(file => require('./' + srcDirectory + '/' + file).default);
+}
+
+function getAllPossibleLanguages() {
+  return getAllConfigs().map(config => config.availableLanguages)
     .reduce((languages, languages2) => languages.concat(languages2))
     .filter((language, position, languages) => languages.indexOf(language) === position);
 }
 
+function faviconPluginFromConfig(config) {
+  let logo = './static/img/' + config.CONFIG + '-icon.png';
```

Comment:
Should we move the icons from static to under sass, so they are not copied for runtime?